.github/ISSUE_TEMPLATE/bug.yml

```
@@ -12,17 +12,23 @@ body:
           - Posting on [Discourse](https://discuss.python.org/c/users/7)
           - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
           - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
-  - type: checkboxes
+
```

Suggestion for some more subheadings/sections -- these might be ignored but hopefully serve to communicate our messages, without ending up in the issue itself
```suggestion
        **Is this a bug in CPython?**

        Only use this form to report bugs in the CPython project itself. Bugs in third-party projects (e.g. ``pip`` or ``requests``) should be reported to that project instead.

        **Is this issue a duplicate?**

```